terraform/gitops/generate-files/templates/pm4ml/kustomization.yaml.tpl

```diff
@@ -15,4 +15,11 @@ helmCharts:
   version: ${pm4ml_chart_version}
   repo: ${pm4ml_chart_repo}
   valuesFile: values-pm4ml.yaml
-  namespace: ${pm4ml_namespace}
\ No newline at end of file
+  namespace: ${pm4ml_namespace}
+- name: finance-portal # admin-portal
```

Comment:
You can have a look at the alias field. You may want to set it to admin-portal.

Reply:
Seems, there's no `alias` filed for helm chart in _Kustomization_ resource

![image](https://github.com/mojaloop/iac-modules/assets/16463615/c97daf9d-4ba3-41c3-9a4f-aff63416bec8)